cmd/describe/volume.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package describe
+
+import (
+	"github.com/openebs/openebsctl/pkg/cstor/describe"
+	"github.com/openebs/openebsctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeInfoCommandHelpText = `
+This command fetches information and status of the various
+aspects of a cStor Volume such as ISCSI, Controller, and Replica.
+
+#
+$ kubectl openebs describe [pool|volume] [name]
```

Comment:
This should be `$ kubectl openebs describe volume [name] -n [namespace]` from the above text right?